index2.html

```diff
@@ -33,7 +33,7 @@
             </ul>
         </nav>
         
-        <div class="headline">
```

Comment:
- [ ] You have not done anything about fixing the margins of your logo and hamburger menu icon. See the [previous review comment](https://github.com/francksefu/portfolio/pull/1#discussion_r1068803874) for a guide. Please fix this using `margin-left` for the logo and `margin-right` for the hamburger.